cuddly-chameleons/cuddly_chameleons/retro_news/migrations/0003_auto_20200808_1230.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.1 on 2020-08-08 12:30
```

Comment:
It's fine to use Django to generate commits, but you should rename them when they get ugly names like this. `0003_auto_20200808_1230.py` gives me absolutely no clues what this migration does. Autogenerate them, then rename them. They'll still work.